src/table/__integ__/resizable-columns.test.ts

```diff
@@ -178,11 +192,25 @@ describe.each([true, false])('StickyHeader=%s', sticky => {
   );
 
   test(
-    'should render "width: auto" for the last on big screens and explicit value on small',
```

Comment:
Having width="auto" on the sticky header last column technically works because it only makes the column take the full width and the remaining columns have explicitly synced widths anyways. However, it is simpler to just synchronise all column widths by offset value instead.